admin/client/App/screens/Item/actions.js

```diff
@@ -23,15 +23,17 @@ export function selectItem (itemId) {
  */
 export function loadItemData () {
 	return (dispatch, getState) => {
+		const thisLoadRef = {};
```

Comment:
Holy shit this is a genius way of using referential equality! Flabbergasted.

_I'd add a comment above here saying something like "This is staying an empty object, this uses referential equality as an ID mechanism". I got that after reading this diff, but if you're just looking at the code in this file I think it's a bit weird!_